src/microbe_stage/OrganelleDefinition.cs

```diff
@@ -290,9 +299,15 @@ public bool TryGetGraphicsScene(out LoadedSceneWithModelInfo modelInfo)
         return true;
     }
 
-    public bool TryGetCorpseChunkGraphics(out LoadedSceneWithModelInfo modelInfo)
+    public bool TryGetCorpseChunkGraphics(OrganelleUpgrades? upgrades, out LoadedSceneWithModelInfo modelInfo)
     {
-        // TODO: allow this to be affected by upgrades (add as a first parameter to this method)
+        var hasUpgradeGraphics = TryGetGraphicsForUpgrade(upgrades, out var upgradeGraphics);
```

Comment:
This part of this method should have the changes applied I mentioned above for the very similar block of code.